test/models/user_test.rb

```diff
@@ -26,6 +26,12 @@ def setup
     refute User.new(email: test_email).valid?
   end
 
+  test 'email must be saved downcased' do
+    new_email = 'NEW_EMAIL@exaMple.cOm'
+    assert create(:user, email: new_email.downcase)
+    refute User.new(email: new_email).valid?
```

Comment:
@sabrown84 - You can just remove this test, in favor of the one(s) I described above.